CondTools/Hcal/interface/HcalPFCutsHandler.h

```diff
@@ -0,0 +1,33 @@
+#ifndef CondTools_Hcal_HcalPFCutsHandler_h
+#define CondTools_Hcal_HcalPFCutsHandler_h
+
+#include <string>
+#include <iostream>
+#include <typeinfo>
+#include <fstream>
+
+#include "CalibCalorimetry/HcalAlgos/interface/HcalDbASCIIIO.h"
+#include "CondCore/PopCon/interface/PopConSourceHandler.h"
+#include "CondFormats/DataRecord/interface/HcalPFCutsRcd.h"
```

Comment:
```suggestion
```
This does not seem to be needed
On the other hand, includes are missing for:
- FileInPath
- ParameterSet
- string